### ISSUE TITLE
Bake navmesh doesn't do anything

### ISSUE THREAD

**Issue by Favkis** [MilosLukic/Godot-Navigation-Lite#10]
https://user-images.githubusercontent.com/46406204/104119701-4a094300-5342-11eb-8c04-b9bf34cab1b7.mp4

It just says two errors and that's it. 
I did as in tutorial: 
changed Navigation to DetourNavigation (weirdly godot says it's spatial)
changed NavigationMesh to spatial
pressed Create navmesh in Navigation
it created DetourNavigationMesh
pressed Bake Navmesh and result is on video

**Comment by Favkis** [comment]
I fixed it by generatic staticbody collision mesh for different model. If it's just flat bake wont do anything

**Comment by MilosLukic** [comment]
@Favkis yes, this addon only works with collisions. So you have to generate static collision mesh for your object.

Because of editor limitations Godot always assumes nodes are spatials. But they have the gdnative script attached, so thats how you can know its the detour object (and by your custom name of course).

**Comment by Ramshell** [comment]
@MilosLukic Hi there! Any hint on where should I look to see if I can collect the static bodies from Gridmap? I'd be more than happy to try to add support to Gridmaps.
